datadog_checks_dev/setup.py

```diff
@@ -31,7 +31,6 @@
     'pytest-mock',
     'requests>=2.20.0',
     'six',
-    'mock>=2.0.0',
```

Comment:
I added this cause py27 build was failing (also I was importing pytest.mock, not just mock)

Reply:
It's already up top

Reply:
Dont know why it was failing then 🤷‍♀